cli/server/getDatasetHelpers.js

```diff
@@ -109,12 +109,15 @@ const sendJson = async (res, info) => {
     const tree = await utils.readFilePromise(info.address.tree);
     /* v1 JSONs don't define a tree name, so we try to guess it here */
     const mainTreeName = info.parts.join('/');
-    const v2Json = convertFromV1({tree, meta, treeName: mainTreeName});
+    const v2Json = convertFromV1({tree, meta});
     return res.json(v2Json)
   }
 }
 
 const findAvailableTangleTreeOptions = (currentDatasetUrl, availableDatasetUrls) => {
+  const expectedDifferences = new Set(["HA", "NA", "PB1", "PB2", "PA", "NP", "NS", "MP", "L", "S",
```

Comment:
Hey @joverlee521 -- we don't want this for the auspice server (see https://github.com/nextstrain/auspice/pull/792#issuecomment-532606922). Having all the options displayed for the auspice server is fine!

For the nextstrain.org server you can use the manifest JSON (see comment in original issue) to work out which options to display.